github_review_helper_suite_test.go

```diff
@@ -187,3 +200,34 @@ var PullRequestEvent = func(action, headSHA string, headRepository grh.Repositor
   }
 }`
 }
+
+var createStatusEvent = func(sha, state string, branches []grh.Branch) string {
+	branchSHAs := make([]string, len(branches))
+	for i, branch := range branches {
+		branchSHAs[i] = branch.SHA
+	}
+	return `{
+  "id": 214015194,
```

Comment:
I'll rather remove, because JSON doesn't support comments, and concatenating a string around a comment gets messy.